frontend/src/metabase/query_builder/components/view/ViewHeader.jsx

```diff
@@ -178,31 +177,33 @@ function SavedQuestionLeftSide(props) {
     isAdditionalInfoVisible,
     isShowingQuestionDetailsSidebar,
     onOpenQuestionInfo,
-    onOpenModal,
+    onSave,
   } = props;
 
   const hasLastEditInfo = question.lastEditInfo() != null;
 
-  const onHeaderClick = useCallback(() => {
-    onOpenModal(MODAL_TYPES.EDIT);
-  }, [onOpenModal]);
+  const onHeaderChange = useCallback(
+    name => {
```

Comment:
Should we modify `EditableText` in a way that `onChange` is not called if the text is not really changed?

Reply:
That's a good call. I was doing that in the handlers, but it can easily be done here as well